plutus-benchmark/script-contexts/src/PlutusBenchmark/V2/Data/ScriptContexts.hs

```diff
@@ -1,7 +1,9 @@
 {-# LANGUAGE BangPatterns      #-}
 {-# LANGUAGE DataKinds         #-}
+{-# LANGUAGE NamedFieldPuns    #-}
 {-# LANGUAGE OverloadedStrings #-}
 {-# LANGUAGE TemplateHaskell   #-}
+{-# LANGUAGE TypeApplications  #-}
```

Comment:
I don't understand why Data-backed Marlowe is slightly faster now. The only relationship I can think of is that maybe it uses code from this module and adding these two extensions somehow made the compiler generate code which is slightly more optimal?

Reply:
After talking to @zliu41, and seeing that other tests have also been impacted, it's most likely from the two `member'` and `insert'` functions being exported, but it remains to be seen what exactly is going on with the Haskell Core. Anyway, we're lucky that the performance changes work in our favor.